src/main/resources/feedbackQuestionConstSumResultStatsTemplate.html

```diff
@@ -18,12 +18,16 @@
                         <td class="button-sort-none" id="button_sortname" onclick="toggleSort(this);">
                             Points Received
                             <span class="icon-sort unsorted"></span>
-                        </td>
+                        </td>						
+                        <td class="button-sort-none" id="button_sorttotal" onclick="toggleSort(this);">
+						    Total Points 
+						    <span class="icon-sort unsorted"></span>
+				        </td>
                         <td class="button-sort-none" id="button_sortclaimed" onclick="toggleSort(this);" style="width:25%;">
                             Average Points
                             <span class="icon-sort unsorted"></span>
                         </td>
-                    </tr>
+				    </tr>
```

Comment:
Quite a lot of tabs still exist in this file. Can you change them to appropriate whitespaces?